samples/apps/github-qna-webapp-react/src/components/QnA.tsx

```diff
@@ -159,7 +161,18 @@ const QnA: FC<IData> = ({ uri, project, branch, keyConfig, onBack }) => {
                         <Slider min={0} max={1} step={0.1} value={relevance} onChange={onSliderChange} />
                         <Label size="small">1</Label>
                     </div>
-                    <Label size="medium">Relevance: {relevance}</Label>
+                    <InfoLabel
+                        info={
+                            <div style={{ maxWidth: 250 }}>
+                                'Relevance' is used in memory search and is a measure of the relevance score from 0.0 to
+                                1.0, where 1.0 means a perfect match. We encourage users to experiment with different
+                                values.
```

Comment:
do you think it makes sense to give them an indication of why we encourage experimentation / what results they should expect?  They'll get different hits in memory, driving different prompt structure, and different completion I think.  This might be too much for a tooltip though.